badge_server/test_badges.py

```diff
@@ -98,6 +105,26 @@
     status=compatibility_store.Status.SUCCESS,
     timestamp=datetime.datetime.utcnow())
 
+APACHE_BEAM_GIT_GOOGLE_API_CORE_RECENT_INSTALL_ERROR_3 = compatibility_store.CompatibilityResult(
+    [package.Package('git+git://github.com/google/apache-beam.git'),
+     package.Package('google-api-core')],
+    python_major_version=3,   # apache-beam does not support Python 3
+    status=compatibility_store.Status.INSTALL_ERROR,
+    timestamp=datetime.datetime.utcnow())
+
+APACHE_BEAM_GIT_GOOGLE_API_CORE_GIT_RECENT_SUCCESS_2 = compatibility_store.CompatibilityResult(
+    [package.Package('git+git://github.com/google/apache-beam.git'),
```

Comment:
We never do this check, right? I thought that we never check git against git?

Reply:
removed